requirements.txt

```diff
@@ -3,7 +3,7 @@ jaxlib>=0.4.30
 orbax-checkpoint>=0.5.20
 absl-py
 array-record
-aqtp
+aqtp==0.7.4
```

Comment:
Is this necessary? It wans't in the previous PR? At the very least can we use >=0.7.4 instead of equals?

Reply:
Needed as some recent recent aqt changes are breaking the code. Resolving this with aqt team and will remove this requirement thereafter.